ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.html

```diff
@@ -52,10 +52,14 @@
 		>Passenger</mat-button-toggle
 	>
 	<mat-button-toggle
-		[ngClass]="{active: activeDriverButton}"
+		[ngClass]="{
+			active: activeDriverButton,
+			disabled: !currentUser.cars.length
+		}"
 		class="role-chooser-btn"
```

Comment:
pls use ngClass or [disabled], not both of them

Reply:
removed both